mobile/lib/features/trade/channel_creation_flow/channel_configuration_screen.dart

```diff
@@ -0,0 +1,512 @@
+                                      fundWithWalletEnabled =
+                                          maxUsableOnChainBalance.sats >= ownTotalCollateral.sats;
+                                      if (!fundWithWalletEnabled) {
+                                        useInnerWallet = false;
+                                      }
+
+                                      updateCounterpartyCollateral();
+                                    });
+                                  },
+                                ),
+                              ),
+                            ),
+                            Padding(
+                              padding:
+                                  const EdgeInsets.only(top: 15, bottom: 5, left: 10, right: 10),
+                              child: ValueDataRow(
+                                  type: ValueType.amount,
+                                  value: counterpartyCollateral,
+                                  label: 'Win up to'),
+                            )
+                          ],
+                        )),
+                    CustomFramedContainer(
+                        text: 'Order details',
+                        child: Padding(
+                          padding: const EdgeInsets.only(top: 15, bottom: 5, left: 10, right: 10),
+                          child: Column(
+                            mainAxisAlignment: MainAxisAlignment.center,
+                            children: [
+                              ValueDataRow(
+                                  type: ValueType.fiat,
+                                  value: quantity.asDouble(),
+                                  label: 'Quantity'),
+                              ValueDataRow(
+                                  type: ValueType.text,
+                                  value: leverage.formatted(),
+                                  label: 'Leverage'),
+                              ValueDataRow(
+                                  type: ValueType.fiat,
+                                  value: liquidationPrice,
+                                  label: 'Liquidation price'),
+                            ],
+                          ),
+                        )),
+                    CustomFramedContainer(
+                        text: 'Order cost',
+                        child: Padding(
+                          padding: const EdgeInsets.only(top: 15, bottom: 5, left: 10, right: 10),
+                          child: Column(
+                            mainAxisAlignment: MainAxisAlignment.center,
+                            children: [
+                              ValueDataRow(
+                                  type: ValueType.amount,
+                                  value: ownTotalCollateral,
+                                  label: 'Channel size'),
+                              FeeExpansionTile(
+                                  label: "Fee*",
```

Comment:
🔧 The asterix isn't pointing to anything.

Reply:
Fair, I wanted to indicate that one can click on it.